<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YsYfInvoiceStatForm</class>
 <widget class="QWidget" name="YsYfInvoiceStatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>应收应付发票统计</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>应收</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="twIncome">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>月份</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>发票号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>账面金额</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>税金</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>外币金额</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>状态</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>客户</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>应付</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="twCost">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>月份</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>发票号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>账面金额</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>税金</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>外币金额</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>状态</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>客户</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>错误信息</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="twError">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>凭证号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>分录序号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>错误信息</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actDel">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除选定记录</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>twIncome</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>YsYfInvoiceStatForm</receiver>
   <slot>doubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twCost</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>YsYfInvoiceStatForm</receiver>
   <slot>doubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twError</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>YsYfInvoiceStatForm</receiver>
   <slot>doubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twIncome</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>YsYfInvoiceStatForm</receiver>
   <slot>contextMenuRequested(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twCost</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>YsYfInvoiceStatForm</receiver>
   <slot>contextMenuRequested(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>723</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doubleClicked(QModelIndex)</slot>
  <slot>contextMenuRequested(QPoint)</slot>
 </slots>
</ui>
